app/src/main/java/de/westnordost/streetcomplete/util/KryoSerializer.java

```diff
@@ -62,7 +63,8 @@
 			WeekdaysTimesRow.class,
 			OsmLatLon.class,
 			SplitAtPoint.class,
-			SplitAtLinePosition.class
+			SplitAtLinePosition.class,
+			Name.class,
```

Comment:
What is this?

Reply:
I think this is left-over from when the AddAddressStreetForm had it's own serializer, before I moved it back to using the AAddLocalizedName Form. I've removed if from here now.